shared/teams/add-members-wizard/add-phone.tsx

```diff
@@ -39,7 +39,8 @@ const AddPhone = () => {
           ? dispatch(
               TeamsGen.createAddMembersWizardPushMembers({
                 members: r.map(m => ({
-                  assertion: m.foundUser ? `${m.username}+${m.assertion}` : m.assertion,
+                  assertion: m.foundUser ? m.username : m.assertion,
+                  note: m.foundUser ? m.assertionValue : undefined,
```

Comment:
do we have a nicer formatted phone number here?

Reply:
No. I could pass it through libphonenumber if it doesn't contain an `@` though.